### PR TITLE
Make WebTorrent process easier to debug

### PR DESCRIPTION
**Pull request by dcposch**
Add `client` (the WebTorrent client) and `server` (the WebTorrent-to-HTTP streaming server) to the window, so it's available in Dev Tools.

### CODE REVIEW

**Comment by dcposch** [comment]
PS if merging plz use the squash option to it doesn't make so many merge commits

File: renderer/webtorrent.js [hunk 1]
```diff
@@ -244,7 +244,6 @@ function getTorrentProgress () {
 }
 
 function startServer (infoHash, index) {
-  if (server) return
```
Comment (feross) [inline renderer/webtorrent.js]: Won't this leak a server that isn't destroyed yet?

Comment (dcposch) [inline renderer/webtorrent.js]: It was duplicated with the same check in `startServerFromReadyTorrent`

Comment (dcposch) [inline renderer/webtorrent.js]: (The way it was before could actually leak servers. If you get to `startServer` twice on torrents that aren't ready, then both will pass the `if (server)` check, wait for their respective torrents to become ready, then start two servers, leaking one of them.)

**Pull request merged**